megamek/src/megamek/common/weapons/infantry/InfantryWeaponHandler.java

```diff
@@ -157,12 +157,15 @@ protected int calcnClusterAero(Entity entityTarget) {
     }
 
     protected int calcAttackValue() {
-        int av = 0;
+        int av;
         //Sigh, another rules oversight - nobody bothered to figure this out
         //To be consistent with other cluster weapons we will assume 60% hit
-        if ((ae instanceof Infantry) && !(ae instanceof BattleArmor)) {
+        if (ae.isConventionalInfantry()) {
             double damage = ((Infantry) ae).getDamagePerTrooper();
             av = (int) Math.round(damage * 0.6 * ((Infantry) ae).getShootingStrength());
+        } else {
+            // Small fixed wing support
+            av = super.calcAttackValue();
```

Comment:
I hate to ask, but what about battle armor?

Reply:
I'm not really sure what that code is about, though I have some ideas. It's beyond the scope of what I'm working on so I made it more concise but otherwise left it alone.

Reply:
Ah, I think I see what you're saying - that check before was explicitly "is this infantry but not battle armor".